app/models/activity.js

```diff
@@ -99,5 +99,23 @@ export default Model.extend({
   @computed('activeBlock')
   active(block) {
     return Boolean(block && block.get('from'))
+  },
+
+  /**
+   * The name of the activity
+   *
+   * Build as a path "customer > project > task"
+   *
+   * @property name
+   * @type {String}
+   * @public
+   */
+  @computed('task')
+  name(task) {
+    let taskName = task.get('name')
+    let projectName = task.get('project.name')
+    let customerName = task.get('project.customer.name')
+
+    return `${customerName} > ${projectName} > ${taskName}`
```

Comment:
What happens if the task isn't set? This is theoretically not possible but I'd still handle that case somehow..